tests/estimators/speech_recognition/test_pytorch_deep_speech.py

```diff
@@ -121,6 +121,8 @@ def test_all(self, _test_all):
     @pytest.fixture(params=[False, True])
     def _test_all(self, request, setup_class):
         # Only import if deep speech module is available
+        import torch
```

Comment:
this test is missing the default `try / except ARTTestException as e: art_warning(e)` same as above